Sources/TokamakCLI/main.swift

```diff
@@ -7,11 +7,29 @@
 
 import Foundation
 import TokamakLint
+import SwiftCLI
 
-do {
-  let srcRoot = ProcessInfo.processInfo.environment["SRCROOT"]!
-  try lintFolder("\(srcRoot)/Sources/Tokamak/Components/Host")
-} catch {
-  print("Can't lint folder")
-  print(error)
+class LintCommand: Command {
+    let name = "lint"
+    let shortDescription = "Lint folder or file"
+    let path = Parameter()
+
+    func execute() throws {
+        do {
+          let srcRoot = ProcessInfo.processInfo.environment["SRCROOT"]!
+            if path.value.contains(".swift") {
+                try lintFile("\(srcRoot)/\(path.value)")
+            } else {
+                try lintFolder("\(srcRoot)/\(path.value)")
+            }
+        } catch {
+          print("Can't lint folder")
+          print(error)
+        }
+    }
 }
+
+let TokamakCLI = CLI(name: "TokamakCLI", version: "1.0.0", description: "Tokamka CLI tools")
```

Comment:
please set the version to correspond to the current version of the project: 0.1.2